widget/playground/src/components/ExportConfigModal/CodeBlock.tsx

```diff
@@ -20,25 +29,34 @@ SyntaxHighlighter.registerLanguage('javascript', javascript);
 SyntaxHighlighter.registerLanguage('jsx', jsx);
 
 export function CodeBlock(props: CodeBlockProps) {
-  const { language, theme, children } = props;
+  const { language, theme, children, selectedType } = props;
+
+  const [isCopied, handleCopy] = useCopyToClipboard(RESET_INTERVAL);
 
-  const [, handleCopy] = useCopyToClipboard(RESET_INTERVAL);
-  const [open, setOpen] = useState<boolean>(false);
   return (
-    <Tooltip side="bottom" open={open} content={<span> Code Copied! </span>}>
+    <Tooltip
```

Comment:
In the task I see Alert not Tooltip. I think it can be implemented by changing max-height & height of modal and  setting transition for alert transform..